### PR TITLE
Update from Phobos

### PR DESCRIPTION
**Pull request by 9il**

### CODE REVIEW

File: source/mir/ndslice/package.d [hunk 1]
```diff
@@ -66,7 +66,7 @@ assert(matrix(3, 2) == 6); // Math & Fortran index order
 ------
 
 Note:
-In many examples $(LINK2 std_range.html#iota, std.range.iota) is used
+In many examples $(LINK2 mir_ndslice_selection.html#iotaSice, iotaSice) is used
```
Comment (wilzbach) [inline source/mir/ndslice/package.d:69]: typo

File: source/mir/ndslice/selection.d [hunk 2]
```diff
@@ -1526,7 +1521,7 @@ pure nothrow unittest
 {
     import mir.ndslice.slice;
     import std.range: iota;
```
Comment (wilzbach) [inline source/mir/ndslice/selection.d:1523]: remove std.range: iota

File: source/mir/ndslice/selection.d [hunk 3]
```diff
@@ -857,7 +853,7 @@ pure unittest
     import mir.ndslice.iteration: allReversed;
     import std.stdio;
     import std.range: iota;
```
Comment (wilzbach) [inline source/mir/ndslice/selection.d:855]: remove std.range : iota

File: source/mir/ndslice/selection.d [hunk 4]
```diff
@@ -1200,7 +1195,7 @@ pure nothrow unittest
 {
     import mir.ndslice.slice;
     import std.range: iota;
```
Comment (wilzbach) [inline source/mir/ndslice/selection.d:1197]: ditto

File: source/mir/ndslice/selection.d [hunk 5]
```diff
@@ -1311,7 +1306,7 @@ Use $(SUBREF iteration, allReversed) in pipeline before
     import mir.ndslice.slice;
     import mir.ndslice.iteration;
     import std.range: iota, isRandomAccessRange;
```
Comment (wilzbach) [inline source/mir/ndslice/selection.d:1308]: ditto

File: source/mir/ndslice/selection.d [hunk 6]
```diff
@@ -1370,11 +1365,11 @@ unittest
 {
     import std.range: iota;
```
Comment (wilzbach) [inline source/mir/ndslice/selection.d:1366]: ditto

**Comment by wilzbach** [comment]
there are still a couple of unnecessary `std.range: iota` imports.

File: source/mir/ndslice/package.d (hunk 1 above)
Comment (9il) [inline source/mir/ndslice/package.d:69]: This should be fixed in phobos first

File: source/mir/ndslice/selection.d (hunk 2 above)
Comment (9il) [inline source/mir/ndslice/selection.d:1523]: all this changes should be done for Phobos first

**Comment by 9il** [comment]
will make PR for Phobos

**Pull request merged**